src/main/markdown/doc/latest/polymer-tutorial/create.md

```diff
@@ -33,12 +33,14 @@ For the **TodoList** project, we'll need to run `webAppCreator` with the followi
 
       _**Tip**: If you include the GWT SDK folder in your PATH environment variable, you won't have to specify the full path._
 
+      You may have to modify the `pom.xml` before you can run the application. Add `<type>pom</type>` to the gwt dependency, otherwise you will encounter an error. 
 
```

Comment:
This is already documented and explained in : http://www.gwtproject.org/doc/latest/RefCommandLineTools.html#webAppCreator , can you add a link to it?